egs/how2/asr1/conf/tuning/decode_pytorch_transformer.yaml

```diff
@@ -0,0 +1,7 @@
+batchsize: 0
```

Comment:
It seems that the decoding configs of RNN and transformer are very different. If you don't test either of them, you can remove that config.

Reply:
I want to keep this as it is.